components/builder-api-proxy/config/nginx.conf

```diff
@@ -61,14 +61,17 @@ http {
                    '"$http_referer" "$http_user_agent" - $http_x_forwarded_for';
 
   upstream backend {
-    server 127.0.0.1:{{bind.http.first.cfg.port}};
```

Comment:
We do want to keep the singular backend pointing to localhost. Without it, we end up transferring large amounts of data over the network and end up with the performance issues that we saw on the Thursday afternoon/morning in Tokyo after that deploy. You were sitting next to me when I fixed this ;)